content/docs/components/serving/tfserving_new.md

```diff
@@ -68,52 +181,82 @@ data:
 kind: Secret
 ```
 
-Enable S3, set url and point to correct Secret
+Then use the following manifest as an example:
 
-```
-MODEL_PATH=s3://kubeflow-models/inception
-ks param set ${MODEL_COMPONENT} modelBasePath ${MODEL_PATH}
-ks param set ${MODEL_COMPONENT} s3Enable true
-ks param set ${MODEL_COMPONENT} s3SecretName secretname
-```
-
-Optionally you can also override default parameters of S3
-
-```
-# S3 region
-ks param set ${MODEL_COMPONENT} s3AwsRegion us-west-1
-
-# Whether or not to use https for S3 connections
-ks param set ${MODEL_COMPONENT} s3UseHttps true
-
-# Whether or not to verify https certificates for S3 connections
-ks param set ${MODEL_COMPONENT} s3VerifySsl true
-
-# URL for your s3-compatible endpoint.
-ks param set ${MODEL_COMPONENT} s3Endpoint s3.us-west-1.amazonaws.com
-```
-
-### Using GPU
-To serve a model with GPU, first make sure your Kubernetes cluster has a GPU node. Then set an additional param:
-```
-ks param set ${MODEL_COMPONENT} numGpus 1
-```
- There is an [example](https://github.com/kubeflow/examples/blob/master/object_detection/tf_serving_gpu.md)
-for serving an object detection model with GPU.
+```yaml
 
-### Deploying
+apiVersion: extensions/v1beta1
```

Comment:
Lets do the same thing for GCP; i.e. in the GCP section have a spec that loads the model from GCS.

The first spec should just be neutral.